src/test/resources/pages/instructorStudentRecords.html

```diff
@@ -112,7 +112,7 @@
                   <td>
                     Ben (
                     <i>
-                      male
+                      MALE
```

Comment:
No problem. I was looking at this issue today, but hadn't come up with a solution. Thanks for the review.